examples/api/test/widgets/actions/action_listener.0_test.dart

```diff
@@ -0,0 +1,50 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/widgets/actions/action_listener.0.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('It should enable the action listener, trigger it and disable it', (WidgetTester tester) async {
```

Comment:
What is "It"?

Reply:
"It" means "the test", "the widgetTester". How would prefer the description to be ?